#!/usr/bin/python3


def raise_exception_msg(test=""):
    raise NameError(test)
